testsuite/cluster-test/src/atomic_histogram.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use crate::tx_emitter::TxStats;
+use std::sync::atomic::{AtomicU64, Ordering};
+
+const HISTOGRAM_CAPACITY: usize = 2048;
+const HISTOGRAM_WIDTH_MILLIS: u64 = 50;
+
+pub struct AtomicHistogram {
+    buckets: Vec<AtomicU64>,
+}
+
+impl AtomicHistogram {
+    pub fn default() -> Self {
```

Comment:
Note that usually you want to implement trait `Default`, instead of just having method with this name (can be useful in other case)